### ISSUE TITLE
Smarter handling for clauses like 'no ICU beds available'

### ISSUE THREAD

**Issue by suranah** [India-Fights-Pandemic/web-app#7]
Right now they get through our set of filters & ends up showing spurious results

**Comment by gaurav-** [comment]
how about a query like this? https://twitter.com/search?q=%20verified%20%20(ICU)%20-%22not%20available%22%20-unavailable%20-%22wanted%22%20-%22not%20verified%22%20-%22unverified%22%20-%22needed%22%20-%22required%22%20-%22need%22%20-%22leads%22%20&src=typed_query&f=live

@suranah @pshrimal21

**Comment by suranah** [comment]
Yes, @gaurav this is close to what we have right now. I think we may also need to add negative queries like `-"no ICU"` or `-"no oxygen"` depending on the context.
